docs/int/quickstart/group/quickstart-group-leader-creator.md

```diff
@@ -68,7 +85,24 @@ You will prepare the configuration file for the distributed key generation cerem
 4. Follow the flow and accept the advisories.
 5. Configure the Cluster 
     - Input the `Cluster Name` & `Cluster Size` (i.e. number of operators in the cluster). The threshold for the cluster to operate sucessfully will update automatically.
-    - ⚠️ Leave the `Non-Operator` toggle off if you are a **Leader** and will participate in the cluster. Turn the toggle on if you are a **Creator** and won't participate.
+    
+    <Tabs groupId="leader-creator">
+      <TabItem value="leader" label="Leader" default>
+    <ul><li>
+    
+    ⚠️ Leave the `Non-Operator` toggle OFF.
+    
+    </li></ul>
+      </TabItem>
+      <TabItem value="creator" label="Creator">
+    <ul><li>
+    
+    ⚠️ Turn the `Non-Operator` toggle ON.
+    
+    </li></ul>
+      </TabItem>
+    </Tabs>
+
```

Comment:
Doesn't let me comment on lines you haven't changed, but there's a typo 5 lines down from here on *transa**c**tion* fees. https://github.com/ObolNetwork/obol-docs/pull/86/files#diff-07d28d8844565e4e1aa5340dcf75bb2b9505ff18349f8b4440e9c6cabc2b3201R110